/**:
  ros__parameters:
    steering_pins: [ 17, 27 ]
    battery_led_pins: [ 5, 6, 16, 20, 21 ]
    battery_led_voltages: [ 2.8, 3.2, 3.4, 3.6 ]
    drive_pwm_frequency: 36
